testing/test_aux.out

```diff
@@ -6,9 +6,12 @@ Validator exit code 0
 Lint checks
 cmd/inbuild exit code 0
 Extended tests
-RESULT skip protocol.bacnet.version
-RESULT skip protocol.bacnet.version
+RESULT skip protocol.bacnet.version Bacnet device not found.
+RESULT skip protocol.bacnet.pic
 RESULT info protocol.bacnet.version Protocol version: 1
+RESULT fail protocol.bacnet.pic
+RESULT info protocol.bacnet.version Protocol version: 1
```

Comment:
@JoelRedstone Are you intentionally running the `info` result of `protocol.bacnet.version` twice?

Reply:
Well yes and no... bacnet.version and bacnet.pics test is performed for each faux device stated in "testing/test_aux.sh".

Once you enable the bacext module in Daq, it will run all bacnet related test for each device. 

https://github.com/JoelRedstone/daq/blob/bacnet_pics/subset/bacnet/test_bacext

In the link above you can see here that all tests are executed one after the other. 
These idea came from Trevor that said he wanted all bacnet test in a single test_*** file rather than having multiple test_*** and Dockerfile.test_*** file for each bacnet test (it would have been the same code in multiple files otherwise).

Reply:
Understood 👌